src/write.c

```diff
@@ -1325,7 +1326,52 @@ static avifResult avifImageCreateAllocate(avifImage ** sampleTransformedImage, c
     return avifImageAllocatePlanes(*sampleTransformedImage, planes);
 }
 
-static avifResult avifEncoderCreateSatoImage(const avifEncoder * encoder,
+// Finds the encoded base image and decodes it.
+// Callers of this function must free *codec and *decodedBaseImage if not null, in case of success or not.
```

Comment:
Nit: Change "in case of success or not" to "whether the function succeeds or not"